TreasureData/TreasureData.m

```diff
@@ -394,6 +413,18 @@ - (void)disableServerSideUploadTimestamp {
     self.serverSideUploadTimestampColumn = nil;
 }
 
+- (void)enableAutoAppendRecordUUID {
+    self.autoAppendRecordUUIDColumn = @"record_uuid";
+}
+
+- (void)enableAutoAppendRecordUUID: (NSString*)columnName {
+    self.autoAppendRecordUUIDColumn = columnName;
```

Comment:
I see. 
It sounds a little counterintuitive (too geeky) but no big deal as long as it's functionality is as expected.